__tests__/lib/fetch.test.ts

```diff
@@ -21,6 +21,7 @@ describe('#fetch()', () => {
       process.env.GITHUB_RUN_ID = '1658821493';
       process.env.GITHUB_RUN_NUMBER = '3';
       process.env.GITHUB_SHA = 'ffac537e6cbbf934b08745a378932722df287a53';
+      process.env.TEST_RDME_CI = 'true';
```

Comment:
In tweaking the logic for `isGHA`, I had to backfill this to get the test passing again

Reply:
edit, jk I guess we don't? nothing makes sense anymore lol https://github.com/readmeio/rdme/pull/739/commits/04538a896627b8a49a9c92d9ece99fbc4b16c0fb